spec/index.bs

```diff
@@ -6051,9 +6088,9 @@ called the render pass encoder can no longer be used.
                     error and stop.
                     <div class=validusage>
                         - |this|.{{GPUProgrammablePassEncoder/[[debug_group_stack]]}}.length is 0.
-                    </div>
 
-                Issue: Add remaining validation.
+                        Issue: Add remaining validation.
+                    </div>
```

Comment:
Still needs TODO to enqueue the attachment stores (with storeOp clear); should add TODO in beginRenderPass too for attachment loads (with loadOp clear)

Reply:
Done. Added an `Issue:` for each.